src/views/preview/preview.scss

```diff
@@ -671,3 +671,20 @@ $stage-width: 480px;
         }
     }
 }
+
+#view {
+    min-height: 80%;
+}
+
+html, body, #app, .page {
+    height: 100%;
+}
+
+.footer {
+    position: absolute;
+    bottom: 0;
+}
```

Comment:
I'm not sure I fully understand the purpose of these changes, or why they are in this file. Could you say more? It seems to work for me with these commented out, keeping only the change in lines 688-690.

Reply:
Gotcha. I'm not seeing any issues with the 404 page when I remove those lines. Could you test it locally and confirm that?